apps/console/src/features/server/pages/remote-logging-page.tsx

```diff
@@ -87,7 +87,8 @@ export const RemoteLoggingPage: FC<RemoteLoggingPageInterface> = (
                 mutateRemoteLoggingRequest={ mutateRemoteLoggingRequest }
                 logType={ LogType.AUDIT }
                 logConfig={ remoteLogPublishingConfigs?.find(
-                    (config: RemoteLogPublishingConfigurationInterface) => config.logType === LogType.AUDIT
+                    (config: RemoteLogPublishingConfigurationInterface) =>
```

Comment:
Is logType null safe ?

Reply:
Added null check by https://github.com/wso2/identity-apps/pull/5834